tests/36.t

```diff
@@ -0,0 +1,7 @@
+description: delete til cursor
+keys: abc \002 \025 \\n
```

Comment:
I decided to remove the ability to use aliases in 5fc3844 due to the
following reasons:

- Less code

- There's a 1-1 mapping between the keys present in test-files and the
  ones declared in the `get_key` function.

I like the idea of comment support in order to add clarity. Something
simple as:

```sh
val=$(echo $val | sed 's/#.*//')
```

... in `tests/test.sh` will probably do, thoughts?